fedimint-cli/src/lib.rs

```diff
@@ -578,6 +584,38 @@ enum Command {
     /// Show the status according to the `status` endpoint
     Status,
 
+    ConfigDecrypt {
```

Comment:
This will be slightly annoying to use imo since `fedimint-cli` always wants a client datadir afaik. I think we might want to reorganize the commands a bit.